Remove Push Quantum from events

veena-vee989 opened the pull request:
Removing Push Quantum because they just got back to me and told me that they didn't run the event.

nathanshammah commented on participating-projects.md:
```diff
@@ -260,7 +265,7 @@ Pulser is a Python library for programming neutral-atom quantum devices at the p
 QCOR is a quantum-retargetable compiler platform providing language extensions for both C++
 and Python that allows programmers to express quantum code as stand-alone kernel functions.
-
+You can watch the Quantum Software Talk on QCOR and XXAC on [YouTube](https://www.youtube.com/watch?v=GzslhEnHUHI&list=PL-VMs2BCTI_nnSQmBaccJ1CdQfw5cpHSp&index=2).
```
```suggestion
You can watch the Quantum Software Talk on QCOR and XACC on [YouTube](https://www.youtube.com/watch?v=GzslhEnHUHI&list=PL-VMs2BCTI_nnSQmBaccJ1CdQfw5cpHSp&index=2).
```

nathanshammah commented on participating-projects.md:
```diff
@@ -280,6 +285,7 @@ and Python that allows programmers to express quantum code as stand-alone kernel
 XACC is a service-oriented, system-level software infrastructure in C++ promoting an
 extensible API for the typical quantum-classical programming, compilation, and execution
 workflow.
+You can watch the Quantum Software Talk on QCOR and XXAC on [YouTube](https://www.youtube.com/watch?v=GzslhEnHUHI&list=PL-VMs2BCTI_nnSQmBaccJ1CdQfw5cpHSp&index=2).
```
```suggestion
You can watch the Quantum Software Talk on QCOR and XAAC on [YouTube](https://www.youtube.com/watch?v=GzslhEnHUHI&list=PL-VMs2BCTI_nnSQmBaccJ1CdQfw5cpHSp&index=2).
```

nathanshammah commented:
Thanks @crazy4pi314.

veena-vee989 committed:
Update index.njk

Removing Push Quantum because they just got back to me and told me that they didn't run the event.

cosenal reviewed (approved):
LGTM
